filebeat/reader/multiline/multiline.go

```diff
@@ -292,8 +299,10 @@ func (mlr *Reader) addLine(m reader.Message) {
 	maxBytesReached := (mlr.maxBytes <= 0 || space > 0)
 	maxLinesReached := (mlr.maxLines <= 0 || mlr.numLines < mlr.maxLines)
 
+	truncated := mlr.truncated
 	if maxBytesReached && maxLinesReached {
 		if space < 0 || space > len(m.Content) {
+			truncated = space - len(m.Content)
```

Comment:
What happens if `space < 0` here?

Reply:
Nice catch. I think it was a leftover. This value of truncated was never used, because it was only added if `len(m.Content)` was bigger than `space`. However, in this branch `space` equals `len(m.Content)`, so the `truncated` value here was unnecessary.